_layouts/default.html

```diff
@@ -7,7 +7,19 @@
     <link rel="stylesheet" href="/assets/css/styles.css">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
     <script src="https://code.jquery.com/jquery-3.3.1.js" integrity="sha256-2Kok7MbOyxpgUVvAk/HJ2jigOSYS2auK4Pfzbm7uH60=" crossorigin="anonymous"></script>
-    <script>
+      <script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
+      <script>
+          function removeTags(str) {
+              if ((str===null) || (str===''))
+                  return false;
+              else
+                  str = str.toString();
```

Comment:
Pas besoin du else quand tu early return

```suggestion
              if (str === null || str === '') {
                  return false;
              }

              str = str.toString();
```